storage/mysql/tree_storage.go

```diff
@@ -151,7 +151,7 @@ func decodeSignedTimestamp(signedEntryTimestampBytes []byte) (trillian.SignedEnt
 	return signedEntryTimestamp, nil
 }
 
-func encodeSignedTimestamp(signedEntryTimestamp trillian.SignedEntryTimestamp) ([]byte, error) {
+func EncodeSignedTimestamp(signedEntryTimestamp trillian.SignedEntryTimestamp) ([]byte, error) {
```

Comment:
Done. I'll check it out after all the PRs get submitted as they also modify this code.